packages/patternfly-4/react-core/src/components/Tabs/TabContent.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@patternfly/react-styles';
+import { getUniqueId } from '../../helpers/util';
+
+const propTypes = {
+  /** the index of the active tab */
+  activeKey: PropTypes.number,
+  /** content rendered inside the tab content area. */
+  child: PropTypes.node
+};
+
+const defaultProps = {
+  activeKey: 0,
+  child: null
+};
+
+const TabContent = ({ activeKey, child, ...props }) => {
+  const id = getUniqueId();
+  return (
+    <section
+      hidden={child.props.eventKey !== activeKey}
+      className={css(child.props.className)}
```

Comment:
You are missing the pf-c-tab-content class here.
You are also missing "role=tabpanel"